apluslms_roman/backends/__init__.py

```diff
@@ -112,3 +114,9 @@ def verify(self):
 
     def version_info(self):
         pass
+
+    def remap_path(self, path):
```

Comment:
this would be better named as `get_host_path`.

Reply:
I'll switch it's function name with the function `git_host_path()` in utils.py